bin/pre-commit-hook.js

```diff
@@ -17,7 +18,7 @@ const files = execSync( 'git diff --cached --name-only' )
 	.map( ( name ) => name.trim() )
 	.filter( ( name ) => name.endsWith( '.js' ) || name.endsWith( '.jsx' ) );
 
-const lintResult = spawnSync( 'eslint-eslines', [ ...files, '--', '--diff=index' ], {
```

Comment:
This wasn't working before. Can I get confirmation that `--diff=remote` (which is the default flag) is the correct flag for this case? It appears to be working correctly for me now (I've just tried it and it detected a lint error on a staged file), but I want to be sure.